docs/providers/aws/guide/plugins.md

```diff
@@ -510,6 +510,55 @@ This way, if user sets `anotherProp` by mistake to `some-string`, the Framework
 Serverless: Configuration error: functions.someFunc.events[0].yourPluginEvent.anotherProp should be number
 ```
 
+#### `defineFunctionProperties` helper
+
+Let's say your plugin adds support to a new `someProperty` function property. To use this property, a user would need to have `serverless.yaml` file like this:
```

Comment:
Let's refer to `serverless.yml` (not `serverless.yaml`), simply to maintain consistency with other parts of documentation

Reply:
Done